Submitted By: Jim Gifford (jim at linuxfromscratch dot org)
Date: 2003-09-22
Initial Package Version: 0.77
Origin: Mandrake CVS
Description: Fixes linking on PAM

	This will correct the shadow issue on linking.
 
diff -Naur Linux-PAM-0.77.orig/libpam_misc/Makefile Linux-PAM-0.77/libpam_misc/Makefile
--- Linux-PAM-0.77.orig/libpam_misc/Makefile	2002-06-27 04:04:54.000000000 +0000
+++ Linux-PAM-0.77/libpam_misc/Makefile	2003-09-23 04:44:16.000000000 +0000
@@ -38,6 +38,8 @@
 SLIBOBJECTS = $(addprefix static/,$(LIBOBJECTS))
 endif
 
+LINKLIBS += -L../libpam -lpam
+
 # ---------------------------------------------
 ## rules
 
diff -Naur Linux-PAM-0.77.orig/modules/pam_lastlog/Makefile Linux-PAM-0.77/modules/pam_lastlog/Makefile
--- Linux-PAM-0.77.orig/modules/pam_lastlog/Makefile	2001-02-10 22:33:10.000000000 +0000
+++ Linux-PAM-0.77/modules/pam_lastlog/Makefile	2003-09-23 04:44:16.000000000 +0000
@@ -15,5 +15,6 @@
 endif
 
 TITLE=pam_lastlog
+MODULE_SIMPLE_EXTRALIBS=-lutil
 
 include ../Simple.Rules
